src/js/program/media-controller.js

```diff
@@ -116,6 +116,19 @@ export default class MediaController extends Eventable {
         }
     }
 
+    attachEvents() {
+        const { model } = this;
+        this.attached = true;
+        model.set('attached', true);
+        this.eventQueue.empty();
```

Comment:
put comment to make it obvious that we are not using ```flush``` because it is a single stream with single provider.